apps/portals/b2ai.standards/src/pages/OrganizationDetailsPage.tsx

```diff
@@ -77,184 +80,249 @@ function OrgHeaderCard({ id }) {
   )
 }
 
+export default function OrganizationDetailsPage() {
+  const { id } = useGetPortalComponentSearchParams()
+  const colExpressions = [
+    ORG_TABLE_COLUMN_CONSTS.MAIN_ORGANIZATION_JSON,
+    ORG_TABLE_COLUMN_CONSTS.ASSOCIATED_ORGANIZATION_JSON,
+    ORG_TABLE_COLUMN_CONSTS.RELEVANT_STANDARDS_JSON,
+    ORG_TABLE_COLUMN_CONSTS.GOVERNED_STANDARDS_JSON,
+    ORG_TABLE_COLUMN_CONSTS.DATASET_JSON,
+    ORG_TABLE_COLUMN_CONSTS.D4D,
+  ]
+  const queryBundleRequest: QueryBundleRequest =
+    getQueryBundleRequestWithIdFilter(
+      'Organization_denormalized',
+      colExpressions,
+      [id],
+    )
+  const {
+    data = [],
+    error,
+    isLoading,
+  } = useFetchJsonArrayLengths(queryBundleRequest)
+
+  if (error) {
+    return <ErrorBanner error={error} />
+  }
+
+  if (!id) {
+    return <ErrorPage type={SynapseErrorType.NOT_FOUND} gotoPlace={() => {}} />
+  }
+  if (isLoading) {
+    return <div>Loading...</div>
```

Comment:
Consider a more user-friendly loader, such as `LinearProgress` from `@mui/material`
```suggestion
    return <LinearProgress />
```